### PR TITLE
Add design system initial commit

### PR DESCRIPTION
**Pull request by dnewh**
Signed-off-by: Davey Newhall <newhall@bitwise.io>

### CODE REVIEW

**Comment by zapplebee** [comment]
The readme should be updated from the Create React App default to be relevant to the project.

**Comment by zapplebee** [comment]
eslint config looks good.

File: canopy/design-system/src/styles/layout.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
+.margin-s {
```
Comment (zapplebee) [inline canopy/design-system/src/styles/layout.scss]: This should be put into to a function so it is easy to manage changing the values and keep it DRY

File: canopy/design-system/.gitignore [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (vaporos) [inline canopy/design-system/.gitignore]: .gitignore should only be done at the top-level of the repo (as per our convention).

File: canopy/design-system/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (vaporos) [inline canopy/design-system/README.md]: Wrap at <80 columns.

File: canopy/design-system/src/App.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
Comment (vaporos) [inline canopy/design-system/src/App.js]: Need copyright headers on all files.

File: canopy/design-system/yarn.lock [hunk 5]
```diff
@@ -0,0 +1,11387 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (vaporos) [inline canopy/design-system/yarn.lock]: No autogenerated files should be checked into the repo (at least on master).

**Comment by vaporos** [comment]
Do we need to maintain .eslint config in every subdirectory?

**Comment by vaporos** [comment]
Please remove the .keep file in a separate commit.

File: canopy/design-system/yarn.lock (hunk 5 above)
Comment (dnewh) [reply canopy/design-system/yarn.lock]: resolved

File: canopy/design-system/src/App.js (hunk 4 above)
Comment (dnewh) [reply canopy/design-system/src/App.js]: resolved

File: canopy/design-system/README.md (hunk 3 above)
Comment (dnewh) [reply canopy/design-system/README.md]: resolved

File: canopy/design-system/src/styles/layout.scss (hunk 1 above)
Comment (dnewh) [reply canopy/design-system/src/styles/layout.scss]: resolved

**Comment by dnewh** [comment]
> Do we need to maintain .eslint config in every subdirectory?

Yes, they are similar but not exactly the same between the projects and also have dependencies in each project

File: canopy/design-system/README.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
+This project was bootstrapped with
```
Comment (vaporos) [inline canopy/design-system/README.md]: Is this README.md content actually useful? Looks like maybe it was autogenerated and mostly points to facebook URLs and isn't really canopy/design-system specific.

Comment (dnewh) [reply canopy/design-system/README.md]: It is the generic create-react-app README and will get updated later but it does, for now, list the relevant commands to run the app

**Pull request merged**